hw03_grayr2.py

```diff
@@ -0,0 +1,164 @@
+
+# TODO Do not edit this file directly!
+# TODO Instead, create a new file called hw03_username.py and copy this code into it!
+
+#################################################################################
+# Author: Relly Gray
+# Username: grayr2
+#
+# Assignment:
+# Purpose:
+# Google Doc Link: https://docs.google.com/document/d/1UVBNWf08HXY_rivdGX1H_HSzcBGa33XGG7AeElADZLg/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements: ChatGpt
+# I asked chatgpt how it may be possible to turn my shape into a skyscraper and it gave me back this: rell.setpos(-100, 100 - (floor + 1) * floor_height)  # Adjust position for the next floor
+#
+#################################################################################
+
+
+import turtle
+
+rell = turtle.Turtle()
+rell.speed(10)
+
+def skyscraper():
```

Comment:
You're likely running into scoping issues with the turtle `rell`. Pass it in as an input parameter:

```
def skyscraper(rell):
    ...
```